modules/using-aws-cli-create-iam-role-alb-controller.adoc

```diff
@@ -25,21 +25,21 @@ $ cat <<EOF > albo-controller-trust-policy.json
         {
             "Effect": "Allow",
             "Principal": {
-                "Federated": "arn:aws:iam::777777777777:oidc-provider/<oidc-provider-id>" <1>
+                "Federated": "<oidc_arn>" <1>
             },
             "Action": "sts:AssumeRoleWithWebIdentity",
             "Condition": {
                 "StringEquals": {
-                    "<oidc-provider-id>:sub": "system:serviceaccount:aws-load-balancer-operator:aws-load-balancer-controller-cluster" <2>
+                    "<cluster_oidc_endpoint>:sub": "system:serviceaccount:aws-load-balancer-operator:aws-load-balancer-controller-cluster" <2>
                 }
             }
         }
     ]
 }
 EOF
 ----
-<1> Specifies the Amazon Resource Name (ARN) of the identity provider.
-<2> Specifies the service account for the AWS Load Balancer Controller.
+<1> Specifies the Amazon Resource Name (ARN) of the OIDC identity provider, such as `arn:aws:iam::301721915996:oidc-provider/aos-hypershift-ci-oidc-305153.s3.us-east-2.amazonaws.com/hypershift-ci-3051531`.
+<2> Specifies the service account for the AWS Load Balancer Controller. An example of `<cluster_oidc_endpoint>` is `oidc-provider/aos-hypershift-ci-oidc-305153.s3.us-east-2.amazonaws.com/hypershift-ci-3051531`.
```

Comment:
please replace the account number 301721915996 with 777777777777, and cluster_oidc_endpoint with that in the Prerequisites. Sorry, My fault.
```
Specifies the Amazon Resource Name (ARN) of the OIDC identity provider, such as `arn:aws:iam::777777777777:oidc-provider/rh-oidc.s3.us-east-1.amazonaws.com/28292va7ad7mr9r4he1fb09b14t59t4f`.
Specifies the service account for the AWS Load Balancer Controller. An example of `<cluster_oidc_endpoint>` is `rh-oidc.s3.us-east-1.amazonaws.com/28292va7ad7mr9r4he1fb09b14t59t4f`.
```

Reply:
:heavy_check_mark: